lib/tasks/db.rake

```diff
@@ -295,6 +295,292 @@ task 'db:stats' => 'environment' do
   print_table(DB.query_hash(sql))
 end
 
+class TemporaryDB
+  PG_PATH = "/tmp/pg_schema_tmp"
+  PG_CONF = "#{PG_PATH}/postgresql.conf"
+  PG_SOCK_PATH = "#{PG_PATH}/sockets"
+
+  def port_available?(port)
+    TCPServer.open(port).close
+    true
+  rescue Errno::EADDRINUSE
+    false
+  end
+
+  def pg_bin_path
+    return @pg_bin_path if @pg_bin_path
+
+    ["13", "12", "11", "10"].each do |v|
+      bin_path = "/usr/lib/postgresql/#{v}/bin"
+      if File.exist?("#{bin_path}/pg_ctl")
+        @pg_bin_path = bin_path
+        break
+      end
+    end
+    if !@pg_bin_path
+      bin_path = "/Applications/Postgres.app/Contents/Versions/latest/bin"
+      if File.exists?("#{bin_path}/pg_ctl")
+        @pg_bin_path = bin_path
+      end
+    end
```

Comment:
This is probably minor but I wonder if we can use `RUBY_PLATFORM` is detect which platform the rake task is being executed on to figure out where the bin path is.

Reply:
Yeah I am not sure I even want to support this outside of our docker install. Considering removing all the mac stuff here.